.github/workflows/opt-out-inbound-plan.yml

```diff
@@ -0,0 +1,59 @@
+name: Terraform plan for opt-out-inbound
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/opt-out-inbound-plan.yml
+      - terraform/services/opt-out-inbound/**
+  workflow_dispatch: # Allow manual trigger
+
+jobs:
+  check-terraform-fmt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./actions/setup-tfenv-terraform
+      - run: terraform fmt -check -diff -recursive terraform/services/opt-out-inbound
```

Comment:
Error: No file or directory at terraform/services/opt-out-inbound

Reply:
Yep, good check. Those files aren't in the main branch, but they'll be there when I merge it into the new_ab2d_opt-out_lambda branch for work on #11.